dagster/implnets/templates/v1/implnet_ops_SOURCEVAL.py

```diff
@@ -697,6 +703,40 @@ def SOURCEVAL_bucket_urls(context):
     get_dagster_logger().info(f"bucker urls report  returned  {r} ")
     return
 
+@op(ins={"start": In(Nothing)})
+def SOURCEVAL_summarize(context) :
+    source_name = "SOURCEVAL"
+    endpoint = GLEANERIO_SUMMARY_GRAPH_ENDPOINT
+    summary_namespace = GLEANERIO_SUMMARY_GRAPH_NAMESPACE
+
+    try:
+        sumnsgraph = mg(mg.graphFromEndpoint(endpoint), summary_namespace)
+        summaryendpoint = endpointUpdateNamespace(endpoint, summary_namespace)
+
+        summarydf = get_summary4repoSubset(endpoint, source_name)
+
+        nt, g = summaryDF2ttl(summarydf, source_name)  # let's try the new generator
+
+        summaryttl = g.serialize(format='longturtle')
+
+        inserted = sumnsgraph.insert(bytes(summaryttl, 'utf-8'), content_type="application/x-turtle")
+
+        # TO DO: upload to minio
+        if not inserted:
```

Comment:
Don't need this.  do need to raise an exception so the step is flagged as bad.

Reply:
I just removed it and simply raise an exception